### DISCUSSION TITLE
New team member

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#403] [Team Posts]
@Akaflieg-Freiburg/enroute

Dear all,

I would like to welcome Michael Paus to our team! Michael develops our partner software, the [AIP Browser DE](https://mpmediasoft.de/products/AIPBrowserDE/help/AIPBrowserDE.html). We joined our team to simplify communication concerning issues and new features that are relevant for both of our two projects.

Best,

Stefan.

**Comment by mipastgt** [comment]
Many thanks for the invitation to join the team. I hope I can contribute something to the project and share a few ideas with you.

**Reply by TTC70** [reply]
Michael, welcome to the team.

**Comment by Heinrich-Ffm** [comment]
> Many thanks for the invitation to join the team. I hope I can contribute something to the project and share a few ideas with you.

Cool software, thanks for that.
